app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
```

Comment:
add a blank line between imports according to pep8  https://peps.python.org/pep-0008/#imports